api/api.go

```diff
@@ -40,6 +42,7 @@ type Authority interface {
 	GetEncryptedKey(kid string) (string, error)
 	GetRoots() (federation []*x509.Certificate, err error)
 	GetFederation() ([]*x509.Certificate, error)
+	GetDatabase() db.AuthDB
```

Comment:
I don't like to have access to the DB at the `/api` level, if we need to access to the DB we should probably go through a layer like the authority package.

Reply:
We don't need it.